pkgs/development/tools/aws-sam-cli/default.nix

```diff
@@ -26,14 +35,13 @@ let
 in
 
 with py.pkgs;
-
 buildPythonApplication rec {
   pname = "aws-sam-cli";
-  version = "0.16.1";
+  version = "0.19.0";
```

Comment:
https://github.com/awslabs/aws-sam-cli/tree/v0.22.0

It's unfortunate this was open so long for the update to be out of date.
But a successful nudge helped with at least one thing :smile:

Reply:
sigh..

Reply:
busting my chops... lol :rofl: